pkg/scalers/cassandra_scaler.go

```diff
@@ -172,6 +209,13 @@ func newCassandraSession(meta *CassandraMetadata, logger logr.Logger) (*gocql.Se
 		Password: meta.password,
 	}
 
+	if meta.enableTLS {
+		cluster.SslOpts = &gocql.SslOptions{
+			CertPath: meta.cert,
+			KeyPath:  meta.key,
+		}
```

Comment:
Do we want to read the values from the file system? this can enforce cluster admins to update KEDA on each cert change

Reply:
@JorTurFer Yes!! 
gocql module supports certificate path as I could not get around passing the cert and key values as input parameters for the cassandra connection. 
Then later when I took a look at keda org certs and found that certs are read from secret mounted to /cert/ path on keda operator. I took a similar approach to add cassandra certs to the same secret and read the certs from /cert/<cassandra>-cert.pem and /cert/<cassandra>-key.pem

Reply:
Makes sense. The last question about this, does it make sense supporting the CA field too?

Reply:
@JorTurFer It surely does, I am thinking how could I miss that. Thank you for pointing that, let me work on it and enable caPath as well. 
But otherwise to connect to DataStax cassandra, cert and key were something required and it did work. I am surely taking up adding caPath as well.